cmd/ipfs/daemon.go

```diff
@@ -180,7 +182,7 @@ func defaultMux(path string) corehttp.ServeOption {
 
 var fileDescriptorCheck = func() error { return nil }
 
-func daemonFunc(req cmds.Request, res cmds.Response) {
+func daemonFunc(req cmds.Request, re cmds.ResponseEmitter) {
```

Comment:
yeah, but it's only in the places where we use the new API. I feel using `res` would be confusing because there are places we still handle a Response